src/rust/engine/src/core.rs

```diff
@@ -60,11 +60,6 @@ pub struct TypeConstraint(pub Id);
 #[derive(Clone, Copy, Debug, Default, Eq, Hash, PartialEq)]
 pub struct Function(pub Id);
 
-// The name of a field.
```

Comment:
Might consider leaving this behind as `pub type Field = String;` ? I don't feel strongly though.